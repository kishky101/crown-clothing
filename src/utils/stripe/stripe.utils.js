import {loadStripe} from '@stripe/stripe-js';

export const stripePromise = loadStripe(
    "pk_test_51MZbh8Bq8CzzyGMqngg7JKxy4ULszaorrVJnkX4FSdMHI1uc7o7qZbekg99mtuI4Zc8PNRtW1d8qyVdUrQjpRAMq00ndNY1lJi"
);