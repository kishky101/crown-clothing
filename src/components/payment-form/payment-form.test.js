import renderer from 'react-test-renderer';
import PaymentForm from './payment-form.component';
import { MockStore } from '../../utils/test/setup';
import { Elements } from '@stripe/react-stripe-js';
import { stripePromise } from '../../utils/stripe/stripe.utils';

test('creating payment-form snapshot', () => {
    const paymentform = renderer.create(
        <MockStore>
            <Elements stripe={stripePromise}>
                <PaymentForm />
            </Elements>
        </MockStore>
    ).toJSON
    expect(paymentform).toMatchSnapshot();
})