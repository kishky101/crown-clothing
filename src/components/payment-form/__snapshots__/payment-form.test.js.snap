// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating payment-form snapshot 1`] = `[Function]`;
